content/en/blog/_posts/2020-03-16-kubernetes-the-hard-way.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: blog
+title: SIG Scalability - Learning Kubernetes the Hard Way
+date: 2020-03-16
+slug: sig-scalability-learning-kubernetes-the-hard-way
+---
+
+**Authors:** Alex Handy
+
+There is a strain of thinking in the software development community that pushes for the most challenging and rigorous possible method of learning a new language or system. These tend to go by the moniker of "Learn \_\_ the Hard Way." Examples abound: Learn Code the Hard Way, Learn Python the Hard Way, and of course, the one that started the whole movement, Zed Shaw's courses in the topic.
+
+While there are folks out there who offer you a "Learn Kubernetes the Hard Way" type experience (most notably Google's Kelsey Hightower), any "Hard Way" project should attempt to cover every aspect of the core topic's principles.
```

Comment:
```suggestion
While there are folks out there who offer you a "Learn Kubernetes the Hard Way" type experience (most notably [Kelsey Hightower's](https://github.com/kelseyhightower/kubernetes-the-hard-way)), any "Hard Way" project should attempt to cover every aspect of the core topic's principles.
```

Reply:
Suggestion: Link and avoid phrasing a person's individual activity as owned by the company